ui/lib/apps/SlowQuery/components/List.tsx

```diff
@@ -41,6 +44,13 @@ export function getDefSearchOptions(): ISearchOptions {
   }
 }
 
+const defColumnKeys = {
```

Comment:
Declare the type of `defColumnKeys` before it?

Reply:
done!